tests/network/sriov.go

```diff
@@ -182,8 +159,9 @@ var _ = Describe("[Serial]SRIOV", func() {
 			defer tests.UpdateKubeVirtConfigValueAndWait(testsuite.KubeVirtDefaultConfig)
 
 			vmi := newSRIOVVmi([]string{sriovnet1}, defaultCloudInitNetworkData())
-			vmi = startVmi(vmi)
-			vmi = waitVmi(vmi)
+			var err error
+			vmi, err = createAndWaitVMI(vmi)
```

Comment:
nit: Is it possible to use a short variable declaration?
There are several similar cases bellow.

Reply:
Could you please explain what you mean? Perhaps by an example?

Reply:
Is this code snippet possible in this location?
```go
vmi, err := createVMIAndWait(vmi)
```

Reply:
Instead of the declaration of the `err` variable:
```go
var err error
```

Reply:
No, because `vmi` is already declared.

Reply:
Well, I think it will shadow it.
I do not know if that is problematic or not. Most usages do not attempt to do this. If you have some ref about this pattern, we can try and consider it.

Reply:
It is fine that `vmi` already exists, as long as `err` is a new variable that is being introduced.
Please see https://go.dev/ref/spec#Short_variable_declarations .

Reply:
> Unlike regular variable declarations, a short variable declaration may redeclare variables provided they were originally declared earlier in the same block (or the parameter lists if the block is the function body) with the same type, and at least one of the non-[blank](https://go.dev/ref/spec#Blank_identifier) variables is new.

Reply:
Yes, seems like it is ok.
Even if it was a new variable shadowing the original, in this case it would have been fine as the variable is not bounded to another function/block.

I do not really like this odd definition, it causes to many question marks to appear.
But I cannot fight the spec, nor the fact it does work (passes build & linter).

Done